tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/client/NettyConnectorTest.java

```diff
@@ -60,4 +64,35 @@ public void testConnectionTimeoutConfig() throws Exception {
       factory.close();
       locator.close();
    }
+
+   @Test
+   public void testConnectionHttpHeaders() throws Exception {
+      TransportConfiguration transport = new TransportConfiguration(NETTY_CONNECTOR_FACTORY);
+      transport.getParams().put(TransportConstants.HTTP_ENABLED_PROP_NAME, true);
+      transport.getParams().put("netty.http.header.accept", "text/html,application/xhtml+xml,application/xml");
+      transport.getParams().put("netty.http.header.Accept-Encoding", "gzip,deflate");
+      transport.getParams().put("netty.http.header.Accept-Language", "en-us,en;q=0.5");
```

Comment:
Should use the constant from `TransportConstants`.

Reply:
I humbly disagree so that if the constant changes it impacts the tests and thus we know that this change can have a big impact

Reply:
That's fair.